lib/github-interface.js

```diff
@@ -0,0 +1,71 @@
+const rc = require('rc')
+const request = require('request-promise-native')
+
+const config = rc('github', { username: '', token: '' })
+const auth = {
+  user: config.username,
+  pass: config.token
+}
+
+const githubApiBase = 'https://api.github.com'
+const githubHeaders = {
+  'User-Agent': 'tableflip-adventure'
+}
+
+// Adds the given version number as a tag to all of the supplied repos and a prerelease
+// with the supplied message to the first repo in the array.
+// Returns a promise that resolves to the Github API release-creation response
+// For example:
+// prerelease({
+//   repos: ['tableflip/project-frontend', 'tableflip/project-api'],
+//   version: 'v2.0.0',
+//   message: 'Adds notifications and basic admin console'
+// })
+async function prerelease ({ repos, version, message }) {
```

Comment:
Suggestions
* Allow repos to be a string (auto arrayify)
* Validate version is a [prerelease semver](http://semver.org/#spec-item-9) (or at least a semver)